### DISCUSSION TITLE
v3无法删除Onedrive

### DISCUSSION THREAD

**Discussion by KimmyLueng** [AlistGo/alist#2900] [Q&A]
Failed get storage driver: no mount path for an storage is: /OneDrive

**Comment by NovaShen0724** [comment]
请提供复现截图/log/复现链接

**Reply by KimmyLueng** [reply]
![无标题](https://user-images.githubusercontent.com/54270610/210524603-c9b0d7c0-4879-46a6-b698-d50b779a92e7.png)

**Comment by NovaShen0724** [comment]
有log吗
可以尝试restart并清除浏览器缓存？

**Reply by KimmyLueng** [reply]
已经重启过并清理缓存

ERRO[2023-01-04 09:26:31] failed get storage driver: no mount path for an storage is: /OneDrive-Movie 
[GIN] 2023/01/04 - 09:26:31 | 200 |  105.317594ms |    27.45.44.180 | POST     "/api/admin/storage/delete?id=23"

**Reply by NovaShen0724** [reply]
应该是alist配置的数据库读写有问题

**Comment by KimmyLueng** [comment]
备份，卸载，重装，恢复，在后台可以把这个onedrive删掉了
